.happy/terraform/modules/ecs-stack/main.tf

```diff
@@ -147,6 +148,9 @@ module backend_service {
 
 module swipe_sfn {
   source                     = "../swipe-sfn"
+
+  count = local.deployment_stage == "prod" ? 1 : 0
```

Comment:
I'm not 100% confident that this and the sfn_arn check will prevent prod from changing, but hopefully we remove this kill switch in the next day or two anyway and the whole thing goes to prod